doc/user/layouts/shortcodes/create-source/syntax-details.html

```diff
@@ -1,6 +1,7 @@
 {{ $connector := .Get "connector" }}
 {{ $formats := split ( .Get "formats" ) " " }}
 {{ $envelopes := split ( .Get "envelopes" ) " " }}
+{{ $keyConstraint := .Get "keyConstraint" }}
```

Comment:
When is this ever false so far?

Reply:
I was afraid that it might be confusing. The following have it as `false`, because they only support formats that result in a single column (e.g. `data`):

```
json-*.md
postgres.md
text-kafka.md
text-kinesis.md
text-pubnub.md
```

Technically, they all support the syntax though. Should I leave it as is (modulo setting it to false explicitly) or add it for all pages (even though we might not expect any usage)?

Reply:
You could set it to `false` explicitly to be more clear, but feel free to leave as is.

Reply:
Done in 6b32f9ba1.